frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -178,24 +187,21 @@ const NewNetworkTrafficUsage: FC = () => {
     const { isOss } = useUiConfig();
 
     const { locationSettings } = useLocationSettings();
-    const {
-        record,
-        newPeriod,
-        setNewPeriod,
-        toTrafficUsageSum,
-        calculateOverageCost,
-        calculateEstimatedMonthlyCost,
-        endpointsInfo,
-    } = useTrafficDataEstimation();
+
+    const [chartDataSelection, setChartDataSelection] =
+        useState<ChartDataSelection>({
+            grouping: 'daily',
+            month: selectablePeriods[0].key,
+        });
```

Comment:
Turns out all I needed was the selection, so there's no point in using the old big hook.